src/epics/getStoreLocations.epic.js

```diff
@@ -0,0 +1,19 @@
+import { GET_STORES_BY_PRODUCT_IDS } from "../redux/actions"
+
+const LCBO_API_KEY =
```

Comment:
since we have this `LCBO_API_KEY` in two places now, let's create a file for our constants and store the key there